pa-aa-{{cookiecutter.country_iso3}}-{{cookiecutter.shock}}/requirements.txt

```diff
@@ -0,0 +1,5 @@
+{%- if cookiecutter.notebooks_in_analysis  == "y" %}
+jupytext==1.11.1
+notebook==6.4.4

```

Comment:
It might be better to put `jupyter` here instead since it's the [metapackage](https://pypi.org/project/jupyter/) that contains `notebook`, `ipykernel` etc